doc/devel/documenting_mpl.rst

```diff
@@ -43,33 +21,47 @@ guide, developers guide, api reference, and FAQs. The documentation suite is
 built as a single document in order to make the most effective use of cross
 referencing.
 
-.. note::
+Sphinx_ also creates ``.rst`` files that are staged in :file:`doc/api` from
+the docstrings of the classes in the Matplotlib library.  Except for
+:file:`doc/api/api_changes/`, these ``.rst`` are created when the
+documentation is built.
 
-   An exception to this are the directories :file:`examples` and
-   :file:`tutorials`, which exist in the root directory.  These contain Python
-   files that are built by `Sphinx Gallery`_.  When the docs are built,
-   the directories :file:`docs/gallery` and :file:`docs/tutorials`
-   are automatically generated. Do not edit the rst files in :file:docs/gallery
-   and :file:docs/tutorials, as they are rebuilt from the original sources in
-   the root directory.
+Similarly, the contents of :file:`docs/gallery` and :file:`docs/tutorials` are
+generated by the `Sphinx Gallery`_ from the sources in :file:`examples` and
+:file:`tutorials`.  These sources consist of python scripts that have ReST_
+documentation built into their comments.  Again, don't directly edit the
+``.rst`` files in :file:`docs/gallery` and :file:`docs/tutorials` as they are
+regenerated when the documentation are built.
 
+Installing dependencies
+-----------------------
 
-Additional files can be added to the various guides by including their base
-file name (the .rst extension is not necessary) in the table of contents.
-It is also possible to include other documents through the use of an include
-statement, such as::
+The documentation for Matplotlib is generated from reStructuredText (ReST_)
```

Comment:
This part is also changed by #9513, which I'd prefer not to have to rebase again...

Reply:
Cool this can wait for #9513 to be put in.  I’m happy to rebase.